daemon/pod/container.go

```diff
@@ -569,8 +569,13 @@ func (c *Container) describeContainer(cjson *dockertypes.ContainerJSON) (*runv.C
 	}
 
 	if cjson.Config.User != "" {
+		users := strings.Split(cjson.Config.User, ":")
+		if len(users) > 2 {
+			return nil, fmt.Errorf("container %s invalid user group config: %s", cjson.Name, cjson.Config.User)
+		}
 		cdesc.UGI = &runv.UserGroupInfo{
-			User: cjson.Config.User,
+			User:  users[0],
+			Group: users[1],
```

Comment:
I suspect here may lead to panic if `config.user` is something like `"root"` (no group at all)